Improve docs

kellertuer opened the pull request:
This improves a few things in the docs

* [x] external links to Wikipedia or ManifoldsBase or Manopt now have prefix icons
* [x] remove the 📖 prefix that was previously used for a few Wikipedia links 
* [x] `make.jl` is a bit easier to use: If the tutorials do not exist and quarto is not run, a warning is issued and they are excluded from the docs – this makes the docs again “render on default”
* [x] we should write a bit more on the about page, I especially would like to link a few packages using the library.

mateuszbaran commented on tutorials/getstarted.qmd:
```diff
@@ -318,7 +318,7 @@ The same mechanism exists for ``[🔗 `parallel_transport_to(M, p, X, q)`](@extr
 Memory allocation is a [🔗 critical performance issue](https://docs.julialang.org/en/v1/manual/performance-tips/#Measure-performance-with-[@time](@ref)-and-pay-attention-to-memory-allocation) when programming in Julia. To take this into account, `Manifolds.jl` provides special functions to reduce the amount of allocations.
 
-We again look at the [📖 exponential map](https://en.wikip edia.org/wiki/Exponential_map_(Riemannian_geometry)). On a manifold `M` the exponential map needs a point `p` (to start from) and a tangent vector `X`, which can be seen as direction to “walk into” as well as the length to walk into this direction. In `Manifolds.jl` the function can then be called with `q = exp(M, p, X)` (see ``[🔗 `exp(M,  ...
+We again look at the [exponential map](https://en.wikip edia.org/wiki/Exponential_map_(Riemannian_geometry)). On a manifold `M` the exponential map needs a point `p` (to start from) and a tangent vector `X`, which can be seen as direction to “walk into” as well as the length to walk into this direction. In `Manifolds.jl` the function can then be called with `q = exp(M, p, X)` (see ``[🔗 `exp(M, p, ...
```
Should we also remove "🔗"?

kellertuer replied:
Well-spotted. I totally forgot about that idea I used (but also only in this tutorial). I replaced all such occurrences.

kellertuer commented:
I also started on the last point, what I still want to do (but that needs a bit of time):

* mention contributions (larger than fixing typos) by going through our PRs
* maybe mention more package that use Manifolds.jl beyond our own ones.

kellertuer commented:
I went through the PRs and added contributors that contributed a manifold; added Caesar, which was the main package I could google to actually use Manifolds.jl

mateuszbaran commented on docs/src/misc/about.md:
```diff
@@ -1,17 +1,36 @@
-## License
-
-[MIT License](https://github.com/JuliaManifolds/Manifolds.jl/blob/master/LICENSE)
+`Manifolds.jl` was started by [Seth Axen](https://github.com/sethaxen), [Mateusz Baran](https://github.com/mateuszbaran), [Ronny Bergmann](https://github.com/kellertuer), and [Antoine Levitt](https://github.com/antoine-levitt) in 2019, after a very fruitful discussion following the release of the first version of [`Manopt.jl`](https://manoptjl.org/). The goal of `Manifolds.jl` is to provide a lib ...
```
```suggestion
`Manifolds.jl` was started by [Seth Axen](https://github.com/sethaxen), [Mateusz Baran](https://github.com/mateuszbaran), [Ronny Bergmann](https://github.com/kellertuer), and [Antoine Levitt](https://github.com/antoine-levitt) in 2019, after a very fruitful discussion following the release of the first version of [`Manopt.jl`](https://manoptjl.org/). The goal of `Manifolds.jl` is to provide a library of manifolds in Julia. The manifolds are implemented using the [´ManifoldsBase.jl](https://github.com/JuliaManifolds/ManifoldsBase.jl/) interface.
```
